### DISCUSSION TITLE
Any plans to support AT32 F435 FCs?

### DISCUSSION THREAD

**Discussion by Pairan** [iNavFlight/inav#9845] [General]
currently we are on STFM32 Chips ... but the new aT32 stuff is available by now.

https://www.diatone.us/collections/mamba-stack/products/mamba-stack-mk5-f435-ai-icm42688p-flight-controller-55a-6s-8bit-esc-stack-30-5mm-m3?variant=40363516133463

are there any plans on making these kind of FCs useable for INAV?

**Comment by b14ckyy** [comment]
The MCUs are supported. We have the NEUTRONRCF435 for example. https://github.com/iNavFlight/inav/pull/8752
But Diatone is a different story. Afaik they have not the highest interest in INAV support.

**Comment by b14ckyy** [comment]
Moving to Discussion for possible further updates
